apps/web/composables/useStructuredData/useStructuredData.ts

```diff
@@ -25,9 +27,33 @@ export const useStructuredData: useStructuredDataReturn = () => {
    * setSigleItemMeta()
    * ```
    */
-  const setSingleItemMeta: SingleItemMeta = (product: Product, variationId: string, categoryTree: CategoryTreeItem) => {
+  const setSingleItemMeta: SingleItemMeta = async (product: Product, categoryTree: CategoryTreeItem) => {
     state.value.loading = true;
+
+    const { data: productReviews, fetchProductReviews } = useProductReviews(product.variation.id, product.item.id);
+    await fetchProductReviews(product.variation.id, product.item.id);
+
+    const { data: reviewAverage, fetchProductReviewAverage } = useProductReviewAverage(product.variation.id.toString());
+    await fetchProductReviewAverage(product.item.id);
```

Comment:
This data is already fetched on the product page. You should be able to access it from `fetchProductReviewAverage ` without refetching it.